### ISSUE TITLE
Release v1.9

### ISSUE THREAD

**Issue by Timple** [Slamtec/rplidar_ros-release#1]
Could version 1.9 be released for ROS kinetic and melodic?

**Comment by boilerbots** [comment]
I agree, please release the latest because at least for the S1 it handles switching the scan_mode to "DenseBoost".

**Comment by Timple** [comment]
Well by now Noetic also should be targeted :)
@kintzhao
